pkg/apis/application/v1alpha1/types.go

```diff
@@ -110,6 +110,12 @@ type ApplicationSource struct {
 	Directory *ApplicationSourceDirectory `json:"directory,omitempty" protobuf:"bytes,10,opt,name=directory"`
 	// ConfigManagementPlugin holds config management plugin specific options
 	Plugin *ApplicationSourcePlugin `json:"plugin,omitempty" protobuf:"bytes,11,opt,name=plugin"`
+	// Chart is a Helm chart name
+	Chart string `json:"chart,omitempty" protobuf:"bytes,12,opt,name=chart"`
```

Comment:
one thought, what about 
```
spec:
  source:
     helm:
       chart: my-chart
```

Reply:
I like it in general, but spec still looks unclean if we introduce `helm` and keep `path` and `targetRevision`. 

I would deprecate and migration `chart`, `targetRevision` and `path`, `targetRevision` to 

```
  # one of
  source:
     helm:
       chart: my-chart
       version: my-version
    git:
       path: my-path
       revision: my-revision
```

Is it ok to create ticket and migrate both when we are ready?

Reply:
sure, I still think we need more user feedback on this feature as a priority over perfection